.github/workflows/My-CI-Pipeline.yml

```diff
@@ -31,19 +31,21 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}    
       - run: docker build --target production --tag daniphilip/hello-world:prod .
       - run: docker push daniphilip/hello-world:prod
+      - run: curl -dH -X POST ${{secrets.WEBHOOK}}
       
- build-three:
-    name: deploying and running onto heroku
-    runs-on: ubuntu-latest
-    needs: build-two
-    if: github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
-    steps:
-      - uses: actions/checkout@v2
-      - uses: gonuit/heroku-docker-deploy@v1.3.3
-        with:
-          email: daniphilip5@outlook.com
-          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
-          heroku_app_name: corndelm8
+ #REMOVE REFERNCES TO HEROKU
+ #build-three:
+    #name: deploying and running onto heroku
+    #runs-on: ubuntu-latest
+    #needs: build-two
+    #if: github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
+    #steps:
+      #- uses: actions/checkout@v2
+      #- uses: gonuit/heroku-docker-deploy@v1.3.3
+        #with:
+          #email: daniphilip5@outlook.com
+          #Heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          #heroku_app_name: corndelm8
```

Comment:
Let's delete this rather than commenting it out - we can always bring it back with Git if we need to (but I'm also happy to hint that for this course we won't bother 😄 )